src/ui_fsmenu/addons_packager_box.cc

```diff
@@ -308,29 +312,73 @@ CampaignAddOnsPackagerBox::CampaignAddOnsPackagerBox(MainMenu& mainmenu, Panel*
                    _("Tribe"),
                    UI::DropdownType::kTextual,
                    UI::PanelStyle::kFsMenu,
-                   UI::ButtonStyle::kFsMenuSecondary) {
+                   UI::ButtonStyle::kFsMenuSecondary),
+     difficulty_(&desc_tribe_box_, 0, 0, 100, UI::PanelStyle::kFsMenu),
+     short_desc_(&desc_tribe_box_, 0, 0, 50, UI::PanelStyle::kFsMenu) {
 	std::vector<Widelands::TribeBasicInfo> tribeinfos = Widelands::get_all_tribeinfos();
 	for (const Widelands::TribeBasicInfo& tribeinfo : tribeinfos) {
 		tribe_select_.add(tribeinfo.descname, tribeinfo.name, g_image_cache->get(tribeinfo.icon),
 		                  false, tribeinfo.tooltip);
 	}
 	tribe_select_.select(tribeinfos.front().name);
-	tribe_select_.selected.connect([this]() {
-		selected_->set_tribe(tribe_select_.get_selected());
-		modified_();
-	});
+	tribe_select_.selected.connect([this]() { edited(); });
+	difficulty_.changed.connect([this]() { edited(); });
+	short_desc_.changed.connect([this]() { edited(); });
+	maps_box_.set_modified_callback([this]() { edited(); });
+
+	difficulty_.set_tooltip(_("The campaigns difficulty. One word."));
+	short_desc_.set_tooltip(_("Short description, which will be appeded to the difficulty."));
+
+	desc_tribe_box_.add(
+	   new UI::Textarea(&desc_tribe_box_, UI::PanelStyle::kFsMenu,
+	                    UI::FontStyle::kFsMenuInfoPanelHeading, _("Difficulty:"), UI::Align::kRight),
+	   UI::Box::Resizing::kFullSize);
```

Comment:
Using `kFullSize` or `kExpandBoth` for Textareas in a horizontal box causes the text to be vertically off-center. You probably need to use `kAlign`-`kCenter` alignment.